crates/db_schema/src/source/person.rs

```diff
@@ -18,8 +18,10 @@ pub struct Person {
   pub actor_id: DbUrl,
   pub bio: Option<String>,
   pub local: bool,
+  #[serde(skip)]
   pub private_key: Option<String>,
   pub public_key: String,
+  #[serde(skip)]
```

Comment:
The urls like inbox_url, etc can maybe be skipped?

Reply:
Yes and public key as well.

Reply:
Doesn't public key get serialized to be sent to other instances?

Reply:
Yes but with Activitypub structs in crates/apub/src/protocol, not with these api structs here. So no need to include it.